spec/features/deposit_via_hydrus_spec.rb

```diff
@@ -78,7 +78,7 @@
     visit "#{Settings.argo_url}/view/#{item_druid}"
 
     # page does not initially display title, loop until reindexed
-    reload_page_until_timeout!(text: "Stanford, Jane Lathrop #{item_title}: 2000-01-01")
+    reload_page_until_timeout!(text: "Stanford, Jane Lathrop #{item_title}: 2000")
```

Comment:
Was the prior expectation wrong or are we changing the tests to expect the wrong behavior that we currently have?

Reply:
the pub date coming from stanford-mods was year only;  this is NOT that field, so I can undo this change!  Thanks for spotting.

Reply:
@jcoyne ok, reverted this change to match the dor-indexing-app.  :-)